eclair-node/src/test/scala/fr/acinq/eclair/api/ApiServiceSpec.scala

```diff
@@ -170,6 +179,17 @@ class ApiServiceSpec extends AnyFunSuite with ScalatestRouteTest with IdiomaticM
       }
   }
 
+  test("plugin injects its own route") {
```

Comment:
We also need a test for the case where routes clash.

Reply:
Added a test and looks like currently it behaves in best possible way: whenever there is a clash a base API endpoint gets retained instead of being overridden.